app/javascript/packs/application.js

```diff
@@ -18,6 +18,11 @@ import '../src/watch_buttons'
 // const images = require.context('../images', true)
 // const imagePath = (name) => images(name, true)
 
+import "@fortawesome/fontawesome-free/js/all";
+import "@fortawesome/fontawesome-free/css/all";
+var fontAwesome = window.FontAwesome;
+fontAwesome.config.autoReplaceSvg = false;
```

Comment:
Unfortunately, newer fa versions replaces <i> into <svg>, which is causing problems with our codebase.

Reply:
How is that causing problems?